sky/skylet/providers/gcp/constants.py

```diff
@@ -85,6 +91,9 @@
     "compute.disks.create",
     "compute.disks.list",
     "compute.firewalls.create",
+    # TODO(skypilot): some users reported that firewalls.{delete,get} should be
+    # removed if VPC/firewalls are separately set up. It is undesirable for a
+    # normal account to have these permissions.
```

Comment:
We need to be careful as this will affect the ability to open ports.

Reply:
Added a comment.